app/webpacker/components/NewcomerChecks/DuplicateCheckerHeader.jsx

```diff
@@ -1,16 +1,50 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { DateTime } from 'luxon';
 import { Button, Message } from 'semantic-ui-react';
 import { duplicateCheckerJobRunStatuses } from '../../lib/wca-data.js.erb';
 
-export default function DuplicateCheckerHeader({ lastDuplicateCheckerJobRun, run }) {
-  if (
-    lastDuplicateCheckerJobRun.run_status === duplicateCheckerJobRunStatuses.in_progress
-    || lastDuplicateCheckerJobRun.run_status === duplicateCheckerJobRunStatuses.not_started
-  ) {
+const JOB_RUN_DELAY_MINUTES = 10;
```

Comment:
Why is this a good estimate? Can we be sure things will never get worse in the future? Please add a short (1-2 lines) code comment outlining your rationale behind this number.

Reply:
Okay, added a comment.